akka-actor/src/main/resources/reference.conf

```diff
@@ -25,6 +25,9 @@ akka {
   # (akka.actor.ActorSystem.Settings, akka.event.EventStream) parameters.
   logging-filter = "akka.event.DefaultLoggingFilter"
 
+  # Specifies the default loggers dispatcher
+  loggers-dispatcher = "akka.actor.default-dispatcher"
```

Comment:
@johanandren there could be more than one, as I understood from the reference.conf